### PR TITLE
Rely on native color picker, drop JS lib; closes #6584

### LINKED ISSUE #6584: Drop colorpicker lib
Color picker is natively implemented in browsers nowadays.

Maybe should we just remove the spectrum Javascript lib; but I do not know if that can cause issues where used.
Along with standard usage from Html class, this is used in kanban and impact javascript.

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes/no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #6584

### CODE REVIEW

File: css/styles.scss [hunk 1]
```diff
@@ -493,6 +493,11 @@ input:required:valid {
    border: 1px solid rgb(211, 211, 211) !important;
 }
 
+input:not(.submit):not([type=submit]):not([type=reset]):not([type=checkbox]):not([type=radio]):not(.select2-search__field)[type=color] {
```
Comment (cedric-anne) [inline css/styles.scss:496]: Should be sufficient
```suggestion
input[type=color] {
```

Comment (trasher) [reply css/styles.scss:496]: This does not work, another rule has precedence

Comment (trasher) [reply css/styles.scss:496]: (that's what I've tried first ;))

Comment (cedric-anne) [reply css/styles.scss:496]: ok

**Pull request merged**